plugins/views/api/api.js

```diff
@@ -3,16 +3,21 @@ var pluginOb = {},
     request = require('request'),
     Promise = require("bluebird"),
     common = require('../../../api/utils/common.js'),
+    moment = require('moment-timezone'),
     authorize = require('../../../api/utils/authorizer.js'),
     countlyCommon = require('../../../api/lib/countly.common.js'),
     plugins = require('../../pluginManager.js'),
     fetch = require('../../../api/parts/data/fetch.js'),
-    countlyModel = require('../../../api/lib/countly.model.js');
```

Comment:
Need to check if someone uses server side model file and if it is up to date and still working.
https://github.com/Countly/countly-server/blob/master/plugins/views/api/lib/countly.model.js